src/backend/distributed/master/master_node_protocol.c

```diff
@@ -628,13 +628,15 @@ ResolveRelationId(text *relationName)
 
 
 /*
- * GetTableDDLEvents takes in a relationId, and returns the list of DDL commands
- * needed to reconstruct the relation. These DDL commands are all palloced; and
- * include the table's schema definition, optional column storage and statistics
- * definitions, and index and constraint definitions.
+ * GetTableDDLEvents takes in a relationId, forShardCreation flag, and returns
+ * the list of DDL commands needed to reconstruct the relation. The function
+ * generates DDL commands specific to shards when forShardCreation flag is set.
+ * These DDL commands are all palloced; and include the table's schema
+ * definition, optional column storage and statistics definitions, and index
+ * and constraint definitions.
  */
 List *
-GetTableDDLEvents(Oid relationId)
+GetTableDDLEvents(Oid relationId, bool forShardCreation)
```

Comment:
xxxSequenceDefaults is not easily understandable (for me at least).  
I am closer to use ```includeSequenceDefaults```